src/answer/precious.rs

```diff
@@ -256,11 +261,18 @@ fn parse_answer_detailed(
         } else {
             let mut formatted_answer: String = String::new();
             for sub_node in instruction.children() {
+                println!("{:?}", sub_node);
+                let syntax: Option<String> = if let Some(s) = sub_node.attr("class") {
+                    class_string_syntax(s.to_string())
+                } else { None };
```

Comment:
It's inefficient to check `class` attribute in each `sub_node`, we can just fetch when we get \<pre\> or \<code\> element